### PR TITLE
ONEUP-7670: fix job status report

### PR DESCRIPTION
**Pull request by yolch-yolchyan**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ inputs:
   slack-webhook-url:
     description: 'Slack channel webhook URL to report to'
     required: true
+  parent-job-status:
```
Comment (catalin-cretu) [inline action.yml:9]: I think we can call this input `job-status` since this actions does not have a `jobs` field

File: action.yml [hunk 2]
```diff
@@ -58,7 +67,7 @@ runs:
         echo "FAILURE_MESSAGE=$FAILURE_MESSAGE" >> $GITHUB_ENV
 
     - name: Report - success [on master]
-      if: job.status == 'success' && (github.ref == 'refs/heads/master' || inputs.always-notify == 'true')
+      if:  ${{ inputs.parent-job-status }} == 'success' && (github.ref == 'refs/heads/master' || inputs.always-notify == 'true')
```
Comment (catalin-cretu) [inline action.yml]: We should either refer to all inputs using `${{ ... }}` or without the brackets in order to be consistent

**Review by catalin-cretu — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (yolch-yolchyan) [reply action.yml:9]: hmm, I feel that job-status is the current one, the one which is being executed, but the parent-hob-status is the job status which called this job

Comment (catalin-cretu) [reply action.yml:9]: There is only one job.
This is an action.

**Pull request merged**